backend/db_setup/data.indaba-sequelize.sql

```diff
@@ -0,0 +1,1735 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 9.4.5
+-- Dumped by pg_dump version 9.5.1
+
+-- Started on 2017-04-11 13:34:11 EDT
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+-- SET row_security = off;
+
+SET search_path = public, pg_catalog;
+
+--
+-- TOC entry 3965 (class 0 OID 0)
+-- Dependencies: 177
+-- Name: Essences_id_seq; Type: SEQUENCE SET; Schema: public; Owner: indabauser
+--
+
+SELECT pg_catalog.setval('"Essences_id_seq"', 57, true);
+
+
+--
+-- TOC entry 3772 (class 0 OID 1599622)
+-- Dependencies: 176
+-- Data for Name: Essences; Type: TABLE DATA; Schema: public; Owner: indabauser
+--
+
+COPY "Essences" (id, "tableName", name, "fileName", "nameField") FROM stdin;
+23	WorflowSteps	WorflowSteps	worflowSteps	title
+20	Groups	Groups	groups	title
+21	Organizations	Organizations	organizations	name
+22	Tasks	Tasks	tasks	title
+4	Products	Products	products	title
+6	UnitOfAnalysis	UnitOfAnalysis	uoas	name
+5	UnitOfAnalysisType	UnitOfAnalysisType	uoatypes	name
+7	UnitOfAnalysisClassType	UnitOfAnalysisClassType	uoaclasstypes	name
+8	UnitOfAnalysisTag	UnitOfAnalysisTag	uoatags	name
+13	Projects	projects	projects	codeName
+14	Discussions	Discussions	discussions	name
+15	Users	Users	users	email
+24	Notifications	notifications	notifications	body
+25	ProductUOA	productUoa	product_uoa	productId
+26	Indexes	Indexes	indexes	title
+27	Subindexes	Subindexes	subindexes	title
+28	IndexQuestionWeights	IndexQuestionWeights	index_question_weights	type
+29	IndexSubindexWeights	IndexSubindexWeights	index_subindex_weights	type
+36	Workflows	Workflows	workflows	name
+37	WorfklowSteps	WorkflowSteps	workflow_steps	title
+38	WorfklowStepGroups	WorkflowStepGroups	workflow_step_groups	stepId
```

Comment:
Lines 37 thru 38 mispelled. "WorkflowSteps" "WorkflowStepGroups"

Reply:
This is only for comparison purposes. I think this is similar in data.indaba.sql.